src/components/ClickableStyle/ClickableStyle.module.css

```diff
@@ -457,91 +457,24 @@
 .clickable-style--link {
   @mixin textLink;
 
-  padding: 0;
+  border-radius: 0;
   border: 0;
-  text-align: left;
   background-color: transparent;
-  gap: 0.25em; /* 2 */
-}
-
-/**
- * Link brand clickable style
- */
-.clickable-style--link.clickable-style--brand {
-  color: var(--eds-theme-color-button-link-brand);
-
-  &:hover {
-    color: var(--eds-theme-color-button-link-brand-hover);
-  }
-
-  &:active {
-    color: var(--eds-theme-color-button-link-brand-active);
-  }
 }
 
 /**
  * Link neutral clickable style
  */
 .clickable-style--link.clickable-style--neutral {
```

Comment:
This component only has neutral link styles because the mixin can only make the brand status link.